File: CS Workshop/Programming/#1 Hello World/hello.cpp
#include <string>
#include <iostream>

using namespace std;

int main(int argc, char** argv)
{
	// declare a string variable to store the name entered
	string name;
	
	// print a prompt to standard output (the screen)
	cout << "Enter your name and press enter: ";
	
	// read the line the user enters from standard in (the keyboard)
	getline(cin, name);
	
	// print "Hello <name>!"
	cout << "Hello " << name << "!";
	
	// print a newline (like return) for good measure
	cout << endl;
	
	// return zero (means no error occurred)
	return 0;
}

File: CS Workshop/Programming/#1 Hello World/hello.py
def main():
	
	# declare a string variable to store the name entered (default to empty)
	#   (this is optional in python)
	name = ""
	
	# print the prompt and read the name variable from standard input (the keyboard)
	name = raw_input("Enter your name and press enter: ")
	
	# print "Hello <name>!"
	print "Hello " + name + "!"
	
	# return zero (means no error occurred)
	return 0	

# if this script is run directly, run the main function above
if __name__ == "__main__":
    main()

